samples/reactjs-todo/package.json

```diff
@@ -11,36 +11,16 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "@babel/cli": "^7.14.5",
-    "@babel/core": "^7.14.6",
-    "@babel/eslint-parser": "^7.14.7",
-    "@babel/preset-env": "^7.14.5",
-    "@babel/preset-react": "^7.14.5",
-    "@nrwl/web": "^13.3.9",
-    "autoprefixer": "^10.2.6",
-    "babel-loader": "^8.2.2",
+    "@types/react": "17.0.40",
+    "@types/react-dom": "17.0.13",
     "bootstrap": "^5.0.1",
-    "css-loader": "^5.2.6",
-    "eslint": "8.2.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-jsx": "^0.1.0",
-    "eslint-plugin-prettier": "^3.4.0",
-    "eslint-plugin-react": "^7.24.0",
-    "eslint-plugin-react-hooks": "^4.3.0",
-    "local-web-server": "^4.2.1",
-    "mini-css-extract-plugin": "^1.6.0",
-    "postcss": "^8.3.5",
-    "postcss-loader": "^6.1.0",
-    "prettier": "^2.3.2",
-    "sass": "^1.35.1",
-    "sass-loader": "^12.1.0",
-    "shx": "^0.3.3",
-    "style-loader": "^2.0.0",
-    "webpack": "^5.39.0",
-    "webpack-cli": "^4.7.2"
```

Comment:
Whoa, that's a lot of dev dependencies gone O_o

How does the sample app build now? Has Nx just completely taken over?

Reply:
I can explain this more in depth if needed. we leverage an `@nrwl/web` or `@some-community-package/vite` (examples). These packages typically contain things like "executors" and "generators".

The executor is essentially packaging these and setting up the configuration for us. So, the "executor" whether a `nrwl` package or community package is handling this.

The positive here is that we can minimize our need to maintain package dependencies and leverage things like `nx migrate latest` to automagically update the workspace.

Reply:
Thanks for explaining this yesterday Ryan - makes a lot more sense. I am becoming more of a fan of Nx every time we speak! 😄